docs/dev/ios/setup-ios-build-environment.md

```diff
@@ -0,0 +1,41 @@
+# Setup the iOS Build Environment
+
+## Prepare your build environment
+
+1. Install Xcode 11.0
+2. Install [Carthage](https://github.com/Carthage/Carthage): `brew install carthage`
+3. Ensure you have Python 3 installed: `brew install python`
+4. Run `bin/bootstrap.sh` to download dependencies.
+5. (Optional, only required for building on the CLI) Install [xcpretty](https://github.com/xcpretty/xcpretty): `gem install xcpretty`
+
```

Comment:
Also, to build the docs, you need to install `jazzy`, but the headers for building native Ruby extensions don't come with the system Ruby.

I had to do:

```
brew install ruby
```

add `export PATH=/usr/local/opt/ruby/bin:$PATH` so the homebrew ruby takes precedence.  (`brew link` doesn't work for this)

`gem install jazzy`

But maybe there's an easier way?

Reply:
It's also worth doing all this *before* installing `xcpretty` so everything gets into the same homebrew ruby install.

Reply:
Ugh, ruby on macos is so complicated.,